store/tikv/2pc.go

```diff
@@ -666,44 +669,60 @@ func sendTxnHeartBeat(bo *Backoffer, store *tikvStore, primary []byte, startTS,
 	}
 }
 
+// checkAsyncCommit checks if async commit protocol is available for current transaction commit, true is returned if possible.
+func (c *twoPhaseCommitter) checkAsyncCommit(ctx context.Context) bool {
+	const asyncCommitKeysLimit = 256
```

Comment:
do we need to check the total size for "key size is large but key count is less" situation?

Reply:
I think it should be considered. But I haven't think carefully about how to set this limit, maybe some tests are needed.